tests/integration/test_execute_job.py

```diff
@@ -20,47 +20,6 @@
 from .. import assert_
 
 
-@pytest.mark.parametrize(
```

Comment:
SHould we have the equivalent test now that we have changed the logic?

Reply:
This test has been moved to unit test, since it doesn't quite interact with GitHub API originally.